### DISCUSSION TITLE
Spike: Onboarding/ Fresh Install - create profile password - add extensive validation

### DISCUSSION THREAD

**Discussion by J-Son89** [status-im/status-legacy#16670] [General]
Following on from implementing the profile password page.

There is some discussion of whether more more extensive validation is needed. e.g pwnd etc.

Sync with PM and leads etc whether this is still being considered and do the relevant research if so

**Comment by du82** [comment]
I think it would be great to offer the option to not have a password, since I do not want to type in a password every time I open my messaging app or make an action inside of it
